syntax = "proto2";

message SearchActionByOrderReq {
	required int64 order = 1;
}

message SearchActionByOrderResp {
	repeated ObjAction action_array = 1;

	message ObjAction {
		required string action_id = 1;
		required string action_name = 2;
		required int64 execute_order = 3;
		required int64 execute_time = 4;
		required string arguments = 5;
	}
}
